src/style.css

```diff
@@ -1 +1,62 @@
-/* Styles go here */
+html {
+  margin: 0;
+  padding: 0;
```

Comment:
html doesn't have margin and paddings, use specific tags to reset this properties